src/timemanager.cxx

```diff
@@ -100,7 +110,8 @@ void TimeManager::setFpb(double f)
 	int bpm = ( samplerate * 60) / f;
 
 	char buffer [50];
-	sprintf (buffer, "TM, setFpb() %i, bpm = %i", int(f), int(bpm) );
+	snprintf(buffer, sizeof(buffer), "TM, setFpb() %i, bpm = %i",
```

Comment:
This is yet another way to debug something. We need to unify this, but maybe not in this PR. Anyway, a compile time flag to mute these outputs would be nice.

Reply:
I don't see an issue with some prints at runtime (these are RT safe, so no audio dropout issues possible). Certainly not this PR.

Reply:
There is no issue for realtime safety or something like that, but when i want to debug something and there are a lot of messages i dont need getting into my way, it is simply annoying. so i would prefer an option to disable the ones i dont care for at the moment. But this is no reason to not merge and i will fix it when i do my refactoring work.